apps/order.rb

```diff
@@ -0,0 +1,54 @@
+RodaApp.route('order') do |r|
+
+  # POST /api/orders/new
+  r.post 'new' do
+    message = CreateOrder.new(r.params).call
+    PostToSlack.say(:food_bot, message) unless test?
+    status(200)
+  end
+
+  # POST /api/orders/show
+  r.post 'show' do
+    order_list = OrderList.last
+    orders = order_list.show_orders
+    if order_list.closed
```

Comment:
closed method should end with '?'. It would be nice to extract this few lines into some separate service.